qa/rpc-tests/minimaldata.py

```diff
@@ -100,10 +101,10 @@ def build_block(self, parent, transactions=(), nTime=None):
         self.block_heights[block.sha256] = block_height
         return block
 
-    def check_for_no_ban_on_rejected_tx(self, tx, reject_reason):
+    def check_for_ban_on_rejected_tx(self, tx, reject_reason=None):
         """Check we don't trigger a ban when sending a txn that the node rejects."""
         self.p2p.send_txs_and_test(
```

Comment:
-Check we don't trigger
+Check we trigger

Reply:
nice catch will fix

Reply:
done